public/app/plugins/datasource/prometheus/language_utils.ts

```diff
@@ -259,3 +261,82 @@ export function escapeLabelValueInExactSelector(labelValue: string): string {
 export function escapeLabelValueInRegexSelector(labelValue: string): string {
   return escapeLabelValueInExactSelector(escapePrometheusRegexp(labelValue));
 }
+
+const FromPromLikeMap: Record<string, AbstractLabelOperator> = {
```

Comment:
Aren't these basically the same? Couldn't we have just 1 map and use it in both directions? By creating something like `getPromOperator` method that retrieves key by value?

Reply:
Added `invert` to avoid duplication in 658005d8191277a5be3bb6099980de1da0979024